modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -2636,8 +2636,11 @@ function synergywholesaledomains_GetTldPricing(array $params)
                 ->setRenewPrice($extension->renew)
                 ->setTransferPrice($transfer_price)
                 ->setRedemptionFeePrice($extension->redemption)
```

Comment:
Can we please check if the price exists prior to setting otherwise it doesn't seem to be returning 0.00 when it's not present in the response